packages/web/src/components/swap/swap-card-content-detail/SwapCardContentDetail.tsx

```diff
@@ -110,7 +109,7 @@ const SwapCardContentDetail: React.FC<SwapCardContentDetailProps> = ({
                 <span className="swap-rate" onClick={handleSwapRate}>
                   {swapRateDescription}
                 </span>
-                {breakpoint !== DEVICE_TYPE.MOBILE && <span className="exchange-price">{`($${unitSwapPrice})`}</span>}
+                {breakpoint !== DEVICE_TYPE.MOBILE && <span className="exchange-price">{`(≈ $${unitSwapPrice})`}</span>}
```

Comment:
Aren't we using the `formatApproximateUSD` function here?

Reply:
yes, I'll improve the `formatApproximateUSD` function and re-commit.